docs/sources/next/javascript-api/k6-experimental/redis/client/_index.md

```diff
@@ -1,20 +1,198 @@
 ---
 title: 'Client'
-excerpt: 'Client is a Redis client to interact with a Redis server or cluster.'
+excerpt: 'Client is a Redis client to interact with a Redis server, cluster, or sentinel.'
 weight: 10
 weight: 10
 ---
 
 # Client
 
-`Client` is a Redis client to interact with a Redis server or cluster. It exposes a promise-based API, which users can interact with in an asynchronous manner.
+`Client` is a [Redis](https://redis.io) client to interact with a Redis server, sentinel or cluster. It exposes a promise-based API, which users can interact with in an asynchronous manner.
 
-Though the API intends to be thorough and extensive, it does not expose the whole Redis API.
-Instead, the intent is to expose Redis for use cases most appropriate to k6.
+Though the API intends to be thorough and extensive, it does not expose the whole Redis API. Instead, the intent is to expose Redis for use cases most appropriate to k6.
 
-Note that the `Client` is configured through the [`Options`](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/redis/redis-options) object.
+## Usage
 
-## Example
+### Single-node server
+
+As shown in the above example, the simplest way to create a new `Client` instance that connects to a single Redis server is by passing a URL string.
+It must be in the format:
+
+`redis[s]://[[username][:password]@][host][:port][/db-number]`
```

Comment:
````suggestion
```
redis[s]://[[username][:password]@][host][:port][/db-number]
```
````

Small nit, and it might be a bit of a personal preference, I think when we have a line of code, it usually looks better with a code block vs code text the way it's rendered in the grafana docs.